tests/suites/test_suite_ecp.data

```diff
@@ -1234,3 +1234,205 @@ ecp_mod_mul_inv:"0000000000000003fffffffffffffffffffffffffffffffffffffffffffffff
 ecp_mul_inv #48 MBEDTLS_ECP_MOD_SCALAR(MBEDTLS_ECP_DP_CURVE448)
 depends_on:MBEDTLS_ECP_DP_CURVE448_ENABLED
 ecp_mod_mul_inv:"0000000000000003fffffffffffffffffffffffffffffffffffffffffffffffffffffff01243a939d867d7e0a75a8568d4d66de88f3ecc1ad37f91a8f9d7d70":MBEDTLS_ECP_DP_CURVE448:MBEDTLS_ECP_MOD_SCALAR
+
```

Comment:
So, should we return error if we the boundary check fails?   Take SECP192R1 as one example, cause we are representing the big nunber with lims, seems it will not report error itself.

Reply:
No I do not think it should fail or that we should have special test functionality for it ( as if in the .function file).

What I meant was that it could be nice if there are test cases in the .data segment that will be checking those inputs, keeping it consistent with the rest of the framework.

Reply:
Have added the test data which ensure the sum overflow for every named curves.